server/appengine/build.gradle

```diff
@@ -0,0 +1,91 @@
+buildscript {
+    repositories {
+        jcenter()
+        maven {
+            url 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
+    }
+
+   dependencies {
+        classpath 'co.present.rpc:compiler:0.2-SNAPSHOT'
+   }
+}
+
+plugins {
+    id 'java'
+    id 'war'
+}
+
+repositories {
+    maven {
+        url 'https://dl.bintray.com/palantir/releases/'
+    }
+}
+
+apply plugin: 'co.present.rpc.compiler'
+
+compileJava {
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+}
+
+configurations.all {
+    // This is pulled in via appengine-gcs-client. :-(
+    exclude group: 'com.google.guava', module: 'guava-jdk5'
+}
+
+dependencies {
```

Comment:
Per CONTRIBUTING.md please describe the dependencies and justifications.  I recognize all of the deps as widely-used (personally) except your present RPCs, roboslack, and objectify.

Reply:
Present RPC: It's just protocol buffers/JSON over HTTP. There's isn't a widely-adopted alternative that I know of. (GRPC requires HTTP/2, isn't suitable for use from web apps, etc.).

Roboslack: Just Java objects modeling Slack's API.

Objectify: The best way to access Cloud Datastore from Java. Google's own docs cover it extensively.